openedx/core/djangoapps/course_live/serializers.py

```diff
@@ -0,0 +1,143 @@
+"""
+Serializers for course live views.
+"""
+from lti_consumer.models import LtiConfiguration
+from rest_framework import serializers
+
+from .models import AVAILABLE_PROVIDERS, CourseLiveConfiguration
+
+
+class LtiSerializer(serializers.ModelSerializer):
+    """
+    Serialize LtiConfiguration responses
+    """
+    lti_config = serializers.JSONField()
+
+    class Meta:
+        model = LtiConfiguration
+        fields = [
+            'lti_1p1_client_key',
+            'lti_1p1_client_secret',
+            'lti_1p1_launch_url',
+            'version',
+            'lti_config'
+        ]
+        read_only = [
+            'version'
+        ]
+
+    def create(self, validated_data):
+        lti_config = validated_data.pop('lti_config', None)
+        instance = LtiConfiguration()
+        instance.version = 'lti_1p1'
+
+        for key, value in validated_data.items():
+            if key in set(self.Meta.fields).difference(self.Meta.read_only):
+                setattr(instance, key, value)
+
+        pii_sharing_allowed = self.context.get('pii_sharing_allowed', False)
+        instance.lti_config = {
+            "pii_share_username": pii_sharing_allowed,
+            "pii_share_email": pii_sharing_allowed,
+            "additional_parameters": lti_config['additional_parameters']
```

Comment:
I think we also need to set the Zoom email here? Email used for configuring zoom

Reply:
`lti_config['additional_parameters']` contains `custom_instructor_email` attr for that I have added [custom validator as](https://github.com/openedx/edx-platform/pull/30012/files#diff-42f9e5980732c4eefb9b10d6805dc8d450bd62c3711b5a4644cc0b0a98d523a7R30) well.